src/vm/eventpipebuffermanager.cpp

```diff
@@ -710,21 +718,17 @@ void EventPipeBufferManager::SuspendWriteEvent()
     }
     CONTRACTL_END;
 
-    CQuickArrayList<EventPipeThread*> threadList;
+    CQuickArrayList<EventPipeThread *> threadList;
     {
         SpinLockHolder _slh(&m_lock);
-        m_writeEventSuspending.Store(TRUE);
```

Comment:
I'm not sure why you moved the flag to EventPipeSession? What we were talking about the other day was replacing this flag with s_pSessions[session_index], but if we still going to maintain an extra flag it seemed like it should be this one.

If the flag stays where it is now then we'd have to update the comments and probably do some asserts that if you call this method you have also independently set that flag. Without setting that flag this method does not work.